torch_xla/csrc/aten_xla_type.cpp

```diff
@@ -1544,6 +1555,20 @@ at::Tensor XLANativeFunctions::lift_fresh(const at::Tensor& tensor) {
   return at::functionalization::impl::to_functional_tensor(tensor);
 }
 
+std::tuple<at::Tensor, at::Tensor> XLANativeFunctions::linalg_inv_ex(
+    const at::Tensor& self, bool check_errors) {
+  TORCH_LAZY_FN_COUNTER("xla::");
```

Comment:
It looks like we didn't do anything with `check_errors`. An easy fix for now is to fallback to cpu if check_errors is provided. wdyt?
https://pytorch.org/docs/stable/generated/torch.linalg.inv_ex.html

Reply:
Good point, makes sense. According to https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/native/native_functions.yaml#L13160, the default value for `check_errors` is `false`, so I've updated to fallback to cpu when this is set to `true`. Also added a comment explaining this in the code.